src/styles/blocks/variables.scss

```
@@ -0,0 +1,7 @@
+$blue-color: #00acdc;
+$pageFont: Roboto, sans-serif;
+$cardWidth: 200px;
+$contentWidth: 100%;
+$colorBlack: #060b35;
+$colorGray: #616070;
+$colorBlue: #00acdc;
```

Why did you create two variables with the same value?
```suggestion
$blue-color: #00acdc;
$pageFont: Roboto, sans-serif;
$cardWidth: 200px;
$contentWidth: 100%;
$colorBlack: #060b35;
$colorGray: #616070;
```